doc/engineering-reference/src/special-modules-reporting/heat-emissions.tex

```diff
@@ -0,0 +1,124 @@
+           & DX heating coil (air-cooled) & Heating Coil Electric Power / Fuel Consumed + Heating Coil Defrost Electric Power + Heating Coil Crankcase Heater Electric Power – Heating Coil Total Heating Rate
+           & Water to air heat pump       & Emit heat in condensing unit defined in Condenser Equipment
+           & DX VRF cooling /heating coil & Emit heat in condensing unit defined in Variable Refrigerant Flow Equipment
+
+Evaporative Coolers                 & Evaporative Cooler Condenser Pump Electricity Consumption + Cooling Coil Basin Heater Electric Energy + Evaporative Cooler Water Volume * rho * evaporation heat of water (\(\delta H_{we}\))
+Variable Refrigerant Flow Equipment & VRF air-to-air heat pump condensing unit (air-cooled) & VRF Heat Pump Total Cooling Rate + VRF Heat Pump Total Electric Power (cooling mode)<br>VRF Heat Pump Total Heating Electric Power / Fuel Rate - VRF Heat Pump Total Heating Rate (heating mode)
+                                    & VRF air-to-air heat pump condensing unit (evaporative-cooled) & Evaporative Cooler Condenser Pump Electricity Consumption + Cooling Coil Basin Heater Electric Energy + Evaporative Cooler Water Volume * rho * evaporation heat of water (\(\delta H_{we}\))
+                                    & VRF air-to-air heat pump condensing unit (water-cooled) & VRF Heat Pump Condenser Heat Transfer Rate
+Plant Heating and Cooling Equipment & Hot water/Steam boiler & Boiler <Fuel Type> Rate + Boiler Ancillary Electric Energy - Boiler Heating Rate
+                                    & Chiller water-cooled & Emit heat in Condenser Equipment
+                                    & Chiller air-cooled / evap-cooled & Chiller Condenser Heat Transfer Rate
+Condenser Equipment                 & Cooling Tower & Cooling Tower Heat Transfer Rate + Cooling Tower Fan Electric Power
+Water Heaters and Thermal Storage   & Water heater & Water Heater <Fuel Type> Rate - Water Heater Heating Rate
+Zone HVAC Forced Air Units          & Window air conditioner / Packaged terminal air conditioner (PTAC) / Packaged terminal heat pump (PTHP); Energy recovery ventilator (ERV); Unit ventilator/heater; Zone outdoor air unit & <unit type> Total Cooling Rate + <unit type> Electric Energy (cooling mode)<br><unit type> Heating Electric Power / Fuel Rate + <unit type> Electric Power - <unit type> Total Heating Rate (heating mode)
+                                    & Water-to-air heat pump & Emit heat in condensing unit defined in Condenser Equipment
+                                    & Zone evaporative cooler & Emit heat in condensing unit defined in Evaporative Coolers
+                                    & Hybrid Unitary HVAC & System dependent; Calculated according to the equipment specification as above
+\bottomrule
+\end{longtable}
+
+There are several parts of heat rejection that the system are not clear defined, and we do not consider the following part into calculation:
```

Comment:
There are several types of components for which the heat rejection is not clearly defined. The following are not included in the heat emissions report: